runtime/doc/lua.txt

```diff
@@ -2271,65 +2271,88 @@ dirname({file})                                             *vim.fs.dirname()*
     Return: ~
         (string) Parent directory of {file}
 
+
 find({names}, {opts})                                          *vim.fs.find()*
     Find files or directories in the given path.
 
-    Finds any files or directories given in {names} starting from {path}. If
+    Finds any files or directories given in {names} starting from {path} (if given). If
     {upward} is "true" then the search traverses upward through parent
     directories; otherwise, the search traverses downward. Note that downward
     searches are recursive and may search through many directories! If {stop}
-    is non-nil, then the search stops when the directory given in {stop} is
-    reached. The search terminates when {limit} (default 1) matches are found.
-    The search can be narrowed to find only files or or only directories by
-    specifying {type} to be "file" or "directory", respectively.
+    is given, then the search stops when the directory given in {stop} is
+    reached. The search terminates when {limit} (1 if omitted) matches are found.
+    The search can be narrowed to find only files or only directories by
+    specifying {type} to be either "file" or "directory", respectively.
+    
+     Examples: >
+     
+       lua = vim.fs.find("aFileOrDirectoryNextToNeovim")
+       => "C:/Users/jdoe/aFileOrDirectoryNextToNeovim"
+       
+       lua = vim.fs.find("allFilesOrDirectoriesAboveNeovim", {upward = true, limit = math.huge})
+       => "/home/jdoe/allFilesOrDirectoriesAboveNeovim"
+       => "/home/allFilesOrDirectoriesAboveNeovim"
+       
+       lua = vim.fs.find("aFileInHomeDirectory", {path = "/home"})
+       => "/home/jdoe/archive/aFileInHomeDirectory"
```

Comment:
As others have already said, do not include `lua = ` in Lua documentation. The `:lua` command is only needed for evaluating Lua on the command line, but these Lua docs cover scripts too. The `lua = ` just adds noise without providing any informational value.